src/Common/ExtraType.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Common;
```

Comment:
What do you suggest? Best I could come up with is `Common\ValueObject` but I deferred from that seeing as there are similar classes to this one in the `Common` namespace.